### DISCUSSION TITLE
Configuration problem.

### DISCUSSION THREAD

**Discussion by aabramo** [lithops-cloud/lithops#631] [Q&A]
Dear all,

thanks for the GREAT work!

As a matter of facts, I suddenly started to get the following message:

ERROR: Invalid requirement: 'ENVPYTHONPATH=/private/var/folders/0c/wvskzqr15jg5jwx1hnk3zb240000gn/T/lithops/lithops.jobs/22e9ffa81d3669d6d751713988987e93-/modules:$PYTHONPATH'
Hint: It looks like a path. File 'ENVPYTHONPATH=/private/var/folders/0c/wvskzqr15jg5jwx1hnk3zb240000gn/T/lithops/lithops.jobs/22e9ffa81d3669d6d751713988987e93-/modules:$PYTHONPATH' does not exist.

from ALL the simple example / templates I got from the site.

I usually try to sort things out myself, but in this case I'm striving without success!

Any clues?

Thnx!
a.

**Comment by JosepSampe** [comment]
Hi @aabramo, 

Thanks for reporting this. Could you provide more context information? such as; which Lithops version are you using? Operating system? Lithops Mode (Localhost, serverless, standalone)? Compute and storage backends? Which configurations variables do you have activated in the config file?

Some lithops DEBUG executions logs are also welcome.

Thanks

**Comment by aabramo** [comment]
Dear Josep,

thanks for your reply.
It was my fault: I had modified (and forgot) the config file so as to have
a customized runtime (auch!)
I finally traced the typo and now things are working nicely.
I use a Mac with OSX Big Sur and I never rely on standard behaviors... I've
got used to thinking that the problems can arise from the non standard
machine configuration.

Thanks again!
a.

 *..**.....**.....*

*ANTONIO ABRAMO, PH.D.*

Associate Professor of Electronics

DPIA - University of Udine

via delle Scienze, 206 - 33100 Udine, Italy

ph.: +39 0432 558283    fax: +39 0432 558251   mob: +39 320 4365990

e.mail | skype: ***@***.***    web: dpia.uniud.it/abramo
<http://www.dpia.uniud.it/abramo>
